### ISSUE TITLE
lib/components/appbar/gf_appbar.dart:312:57: Error: No named parameter with the name 'nullOk'.

### ISSUE THREAD

**Issue by dineelbathina** [ionicfirebaseapp/getwidget#201]
Working fine in stable version after moving to beta and starting web app getting following error.

**../flutter/.pub-cache/hosted/pub.dartlang.org/getwidget-1.2.2/lib/components/appbar/gf_appbar.dart:312:57: Error: No named parameter with the name
'nullOk'.
    final ScaffoldState scaffold = Scaffold.of(context, nullOk: true);  
                                                        ^^^^^^          
../flutter/packages/flutter/lib/src/material/scaffold.dart:1918:24: Context: Found this candidate, but the arguments don't match.
  static ScaffoldState of(BuildContext context) {**

**Comment by Jailson66** [comment]
is related to this flutter commit https://github.com/flutter/flutter/commit/4fa5fe5f0b1c1161b65d1d9c643ed21c098f5ac8

**Comment by dixyantar** [comment]
Same issue. Surprisingly doesn't go away when I change version to 1.1.3.

**Comment by i5o** [comment]
**Describe the bug**
Can't build after upgrading flutter (beta channel)

**To Reproduce**
Steps to reproduce the behavior:
1. Run `flutter upgrade`
2. Try to run an app that exists with this plugin already

**Expected behavior**
App builds no errors

**Additional context**
Error:
```
Target dart2js failed: Exception: ../../.pub-cache/hosted/pub.dartlang.org/getwidget-1.2.3/lib/components/appbar/gf_appbar.dart:312:57:
Error: No named parameter with the name 'nullOk'.
    final ScaffoldState scaffold = Scaffold.of(context, nullOk: true);
                                                        ^^^^^^
../../flutter/packages/flutter/lib/src/material/scaffold.dart:1918:24:
Info: Found this candidate, but the arguments don't match.
  static ScaffoldState of(BuildContext context) {
                       ^^
Error: Compilation failed.
```
```
ignacio@manzana:~/repo/dash$ flutter doctor -v
[✓] Flutter (Channel beta, 1.24.0-10.2.pre, on Linux, locale en_US.UTF-8)
    • Flutter version 1.24.0-10.2.pre at /home/ignacio/flutter
    • Framework revision 022b333a08 (5 days ago), 2020-11-18 11:35:09 -0800
    • Engine revision 07c1eed46b
    • Dart version 2.12.0 (build 2.12.0-29.10.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /home/ignacio/Android/Sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /home/ignacio/Android/Sdk
    • Java binary at: /home/ignacio/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Android Studio
    • Android Studio at /home/ignacio/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.51.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.16.0

[✓] Connected device (2 available)
    • Web Server (web) • web-server • web-javascript • Flutter Tools
    • Chrome (web)     • chrome     • web-javascript • Google Chrome 87.0.4280.66

• No issues found!
```

**Comment by Bilonik** [comment]
Same

**Comment by Sandipkakadiya** [comment]
This issue has been fixed with latest version 1.2.4
